paper/paper.md

```diff
@@ -126,25 +137,7 @@ callback_lsr1 =
                                                )
 solver_stats = trunk(lsr1_LeNet; callback = callback_lsr1, max_time)
 ```
-
```

Comment:
Add a sentence there, to explain that the same example, using Knet and KnetNLPModels.jl, is available at this link [training documentation](https://jso.dev/KnetNLPModels.jl/stable/LeNet_Training/)

Reply:
do you want me to do this or the other comment fixed this ?